plugin/src/main/java/org/opensearch/ml/utils/RestActionUtils.java

```diff
@@ -309,4 +309,13 @@ public static void wrapListenerToHandleSearchIndexNotFound(Exception e, ActionLi
         }
     }
 
+    public static String getActionTypeFromRestRequest(RestRequest request) {
```

Comment:
Can we add java doc to the public method to explain what's going on here?

Reply:
It will capture the method name "predict" or "batch" from the Url path, so we don't need to enforce to input an "action_type" in the request body. Check the PR description for the batch invoke.

Reply:
right, but 3 years later nobody is going to take a look at the "PR description" but at the code.